src/lib/sw.js

```diff
@@ -192,13 +192,15 @@ workboxRouting.registerRoute(normalMatch, async ({url}) => {
   }
   const partial = await response.json();
 
-  // Our target browsers all don't mind if we just place <title> in the middle of the document.
-  // This is far simpler than trying to find the right place in <head>.
   const meta = partial.offline ? `<meta name="offline" value="true" />` : '';
-  const output = layoutTemplate.replace(
-    '%_CONTENT_REPLACE_%',
-    meta + `<title>${partial.title || ''}</title>` + partial.raw,
-  );
+  const title = `<title>${partial.title || 'web.dev'}</title>`;
+  const rssHref = partial.rss || '/feed.xml';
+  const rssTitle = partial.rss ? `${partial.title} on web.dev` : 'web.dev feed';
+  const rss = `<link rel="alternate" href="${rssHref}" type="application/atom+xml" data-title="${rssTitle}"></link>`;
+
+  const output = layoutTemplate
+    .replace('%_HEAD_REPLACE_%', meta + title + rss)
```

Comment:
I honestly did only one replacement out of laziness and the fact that I couldn't find any browser we supported that cared _where_ header-like things existed on the page.

I thought if we ever switched to proper 'streams' it would be faster/easier/simpler to just split the page into "start", "our content", and "end".

I still don't really see a need for things to _actually_ be in the `<head>`, but there's nothing wrong with this approach.